osu.Game/Overlays/Rankings/Tables/UserBasedTable.cs

```diff
@@ -32,7 +33,7 @@ protected UserBasedTable(int page, IReadOnlyList<UserStatistics> rankings)
         protected override Drawable CreateHeader(int index, TableColumn column)
         {
             var title = column?.Header ?? string.Empty;
-            return new UserTableHeaderText(title, HighlightedColumn == title, GradeColumns.Contains(title));
+            return new UserTableHeaderText(title, HighlightedColumn == title, GradeColumns.Contains(title.ToString()));
```

Comment:
These `HighlightedColumn` and `GradeColumns` things are horrifyingly bad - as stated on discord - and need to go away preferably before proper localisation happens. I have some changes that change this to have an actual class hierarchy of the columns and store actual data instead of relying on equality members. Not sure whether that should happen before or after this, though.

Reply:
I think this can still work either way as long as `HighlightedColumn` matches the same localisable string data, but this definitely should be improved as it reads very weirdly.

I think since this would be a blocker to a framework bump it's better to fix the awkward logic after this PR, since this doesn't change anything in the end and it'll still work temporarily for the time being.

Reply:
I think this can be fixed in a follow-up effort.

Reply:
Yeah I'll PR that today, hopefully. Would have possibly even done that yesterday if not for a power cut.